src/main/java/metrics/report_format/JSON.java

```diff
@@ -44,9 +44,19 @@ public String format(CompareDTO dto, List<Double> similarities, List<String> agg
         }
 
         JsonObject aggregationsJson = new JsonObject();
-        Map<String, String> aggregationPairs = getAggregations(dto, aggregations);
+        Map<String, List<Double>> aggregationPairs = getAggregations(dto, aggregations);
         for (String aggregationKey : aggregationPairs.keySet()) {
-            aggregationsJson.addProperty(aggregationKey.toLowerCase(), aggregationPairs.get(aggregationKey));
+            List<Double> aggregationValues = aggregationPairs.get(aggregationKey);
+            StringBuilder data = new StringBuilder();
+            data.append("[ ");
+            for(int i = 0; i < aggregationValues.size(); i++){
+                if(i != 0){
+                    data.append(", ");
+                }
+                data.append(aggregationValues.get(i));
+            }
+            data.append(" ]");
+            aggregationsJson.addProperty(aggregationKey.toLowerCase(), data.toString());
```

Comment:
I'd recommend replacing this code with
```java
aggregationsJson.addProperty(aggregationKey.toLowerCase(), aggregationPairs.get(aggregationKey).toString());
```
And updating the two(?) tests that check these values to not have spaces in the array.

Reply:
makes it look a lot cleaner, thanks